spec/services/workflow_errors_reporter_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+
+RSpec.describe WorkflowErrorsReporter do
+  let(:put) { 'https://sul-lyberservices-test.stanford.edu/workflow/dor/objects/druid:jj925bx9565/workflows/preservationWF/moab-valid' }
+  let(:headers) { { 'Content-Type' => 'application/xml' } }
+  let(:result_array) do
+    { 13 => "Invalid moab, validation error...ential version directories." }
+  end
+  let(:body) { "<process name='moab-valid status='error' errorMessage='{13=>\"Invalid moab, validation error...ential version directories.\"}'/>" }
```

Comment:
you can use previous "let" variables in other "let" definitions ... which could make this more readable (e.g. if you have a var "result_msg" ...)